tests/grid/grids/abaqus/2d/2d_short_handwritten.inp

```diff
@@ -0,0 +1,12 @@
+*HEADING
+This file is the most simplest mesh that can
```

Comment:
`most simplest` -> `simplest`

Reply:
Sorry for the mistake. Fixed it in 5e7b354.